dev/dll/Density/Normal.xaml

```diff
@@ -0,0 +1,127 @@
+            <Thickness x:Key="SliderTopHeaderMargin">0,0,0,8</Thickness>
+            <x:Double x:Key="SliderPreContentMargin">18</x:Double>
+            <x:Double x:Key="SliderPostContentMargin">18</x:Double>
+            <x:Double x:Key="SliderHorizontalHeight">44</x:Double>
+            <x:Double x:Key="SliderVerticalWidth">44</x:Double>
+            <Thickness x:Key="ToggleSwitchTopHeaderMargin">0,0,0,0</Thickness>
+            <x:Double x:Key="ToggleSwitchPreContentMargin">10</x:Double>
+            <x:Double x:Key="ToggleSwitchPostContentMargin">10</x:Double>
+            <Thickness x:Key="TimePickerTopHeaderMargin">0,0,0,8</Thickness>
+            <x:Double x:Key="TimePickerFlyoutPresenterHighlightHeight">44</x:Double>
+            <x:Double x:Key="TimePickerFlyoutPresenterAcceptDismissHostGridHeight">45</x:Double>
+            <Thickness x:Key="DatePickerTopHeaderMargin">0,0,0,8</Thickness>
+            <x:Double x:Key="DatePickerFlyoutPresenterHighlightHeight">44</x:Double>
+            <x:Double x:Key="DatePickerFlyoutPresenterAcceptDismissHostGridHeight">45</x:Double>
+            <Thickness x:Key="AutoSuggestBoxTopHeaderMargin">0,0,0,8</Thickness>
+            <x:Double x:Key="ListViewItemMinHeight">44</x:Double>
+            <Thickness x:Key="AppBarButtonContentViewboxMargin">12,14,0,14</Thickness>
+            <Thickness x:Key="AppBarButtonContentViewboxCompactMargin">0,14,0,14</Thickness>
+            <Thickness x:Key="AppBarButtonContentViewboxCollapsedMargin">0,14,0,4</Thickness>
+            <Thickness x:Key="AppBarButtonOverflowTextTouchMargin">0,11,0,13</Thickness>
+            <Thickness x:Key="AppBarButtonOverflowTextLabelPadding">0,5,0,7</Thickness>
+            <Thickness x:Key="AppBarButtonTextLabelMargin">2,0,2,8</Thickness>
+            <Thickness x:Key="AppBarButtonTextLabelOnRightMargin">8,15,12,17</Thickness>
+            <Thickness x:Key="AppBarToggleButtonOverflowTextTouchMargin">0,11,0,13</Thickness>
+            <Thickness x:Key="AppBarToggleButtonOverflowCheckTouchMargin">12,12,12,12</Thickness>
+            <Thickness x:Key="AppBarToggleButtonOverflowCheckMargin">12,6,12,6</Thickness>
+            <Thickness x:Key="AppBarToggleButtonTextLabelMargin">2,0,2,8</Thickness>
+            <Thickness x:Key="AppBarToggleButtonTextLabelOnRightMargin">8,15,12,17</Thickness>
+            <Thickness x:Key="AppBarToggleButtonOverflowTextLabelPadding">0,5,0,7</Thickness>
+            <Thickness x:Key="ButtonPadding">8,4,8,4</Thickness>
+            <Thickness x:Key="CommandBarMoreButtonMargin">14,23,14,0</Thickness>
+            <x:Double x:Key="AppBarButtonContentHeight">20</x:Double>
+            <x:Double x:Key="AppBarThemeMinHeight">60</x:Double>
+            <x:Double x:Key="AppBarThemeCompactHeight">48</x:Double>
+            <Thickness x:Key="RichEditBoxTopHeaderMargin">0,0,0,8</Thickness>
+        </ResourceDictionary>
+
+        <ResourceDictionary x:Key="HighContrast">
```

Comment:
Most of these values are not theme dependent, which is why I intentionally had them outside the theme dictionaries in generic.xaml. No need to duplicate identical values. Why have you split them out based on theme dictionary here?

Reply:
If I specify below dictionary in app.xaml, the keys not in themeresource part doesn't work as we expected until we change theme. and that's why I split them to themes.
`<ResourceDictionary Source="ms-appx:///Microsoft.UI.Xaml/Density/Normal.xaml" />`

Reply:
Have you looked into why that didn't work? It doesn't seem like the theme should matter for these.